### PR TITLE
(dropbox) update for stable version

### PR DESCRIPTION
**Pull request by RedBaron2**
Update to dropbox  to always have the correct stable version, and to allow for the dropbox-beta package in the future

@AdmiringWorm 
I might have at least lowered the time to **_2 min 52 sec_**

### CODE REVIEW

**Comment by RedBaron2** [comment]
I almost thought something went wrong when I was seeing the install of 29.3.18. It wasn't until I checked the latest beta is 29.3.19. So should I add in this PR a reset of the nuspec file?

**Comment by AdmiringWorm** [comment]
> I almost thought something went wrong when I was seeing the install of 29.3.18. It wasn't until I checked the latest beta is 29.3.19. So should I add in this PR a reset of the nuspec file?

You don't need to do that, because the `update` inside `update.ps1` wasn't uncommented, the package didn't actually update itself. Therefore you still see the previous beta version in the appveyor logs.

**Comment by RedBaron2** [comment]
Changes made. Enjoy the PR

File: automatic/dropbox/update.ps1 [hunk 1]
```diff
@@ -14,26 +14,56 @@ function global:au_SearchReplace {
...
-    $versionRegEx = '.*Dropbox%20([0-9\.]+).*'
-    $downloadUrl = Get-RedirectedUrl $downloadEndpointUrl
-    $version = $downloadUrl -replace $versionRegEx, '$1'
+    $releases = 'https://www.dropboxforum.com/t5/Desktop-client-builds/bd-p/101003016'
+    $HTML = Invoke-WebRequest -UseBasicParsing -Uri $releases
+    $stable_builds = @()
+    $beta_builds = @()
+    $HTML.Links | foreach {
+  	  if ($_.href -match "stable" ) { $stable_builds += $_.href }
+  	  if ($_.href -match "beta" ) { $beta_builds += $_.href }
```
Comment (RedBaron2) [inline automatic/dropbox/update.ps1]: @AdmiringWorm 
I presume you are referring to this but of code? You want me to rename it to like nonstable. So as to not refer to beta? 
This I can work on later, but without it the whole check is pointless.

**Comment by RedBaron2** [comment]
@AdmiringWorm 
Updated update.ps1  to have a update_helper.ps1. This will keep update smaller. Should be ready for your approval 😄

**Pull request merged**